geode-assembly/src/main/dist/LICENSE

```diff
@@ -261,17 +263,18 @@ POSSIBILITY OF SUCH DAMAGE.
 The CDDL Version 1.1 (https://javaee.github.io/glassfish/LICENSE)
 ---------------------------------------------------------------------------
 
-Apache Geode bundles the following file under the Common Development and
+Apache Geode bundles the following files under the Common Development and
 Distribution License:
 
-  - jaxb-istack-commons v2.2 (https://javaee.github.io/jaxb-istack-commons/)
+  - activation v1.1.0
```

Comment:
Something seems wrong that we are bundling both activation and Java.activation.  Can you file a JIRA for followup?

Reply:
filed GEODE-8066 for this